conduct/COC_POLICY.md

```diff
@@ -0,0 +1,109 @@
+This document defines the OpenJS Foundation's code of conduct policy.
+
+If you want to report an incident, go to [code-of-conduct.openjsf.org][CoC].
+
+***
+
+# Code of Conduct Policy
+
+The OpenJS Foundation (OpenJSF) delegates to the Cross Project Council (CPC), via its [charter][], the responsibility of choosing, maintaining, and enforcing a Code of Conduct (CoC) for all of its projects, collab spaces, and for any related activities.
+
+## Code of Conduct
+
+The OpenJSF's code of conduct is located at [code-of-conduct.openjsf.org][CoC]. It is based on the [Contributor Covenant](https://www.contributor-covenant.org/).
+
+A common code of conduct allows a common escalation and appeal process and allows better support of projects in their efforts to apply the code of conduct.
+
+All OpenJSF projects, collab spaces, and any activities related to the OpenJSF are operating under this code of conduct unless otherwise stated. (For example, events organized in partnership with other organizations might be operating under a different code of conduct.)
```

Comment:
How do we dispatch updates of our CoC to project's CoC's? For example, should Node.js's CoC simply redirect to OpenJS's CoC? (https://github.com/nodejs/admin/blob/HEAD/CODE_OF_CONDUCT.md)

Reply:
That's a great point. It seems related to https://github.com/openjs-foundation/cross-project-council/pull/889. Can we address it there, separately from this PR?

Reply:
We already changed the CoC on a number of occasions without a dedicated process, so I don't think we should block on this. Let's not forget that the CPC is composed of representatives of projects precisely so that decisions taken here reflect project needs and don't take anyone by surprise.

Reply:
I have mixed feelings here as for merging this halfway... with open requirements. But not strong enough to oppose to this approach.

My take is that knowing how slow certain pieces of our collaboration process are... We might at some point have an incomplete CoC process/charter for a while until all pieces get finally merged together.

Shouldn't we then merge this to a "feature-branch" and have it all merged to "main" after done?

Reply:
A dedicated process for changing the CoC is not a missing piece, it's an additional feature. The CoC is under the CPC's purview and thus, without any dedicated process for changing the CoC, regular CPC process applies.

Reply:
I see, that's fair and I was under the impression we were changing that. Thx for clarifying!

Sometimes large PR diffs make it confusing to know what's really going on.